packages/@react-spectrum/color/src/Gradients.tsx

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
Missing copyright statement:
```suggestion
/*
 * Copyright 2022 Adobe. All rights reserved.
 * This file is licensed to you under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License. You may obtain a copy
 * of the License at http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software distributed under
 * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
 * OF ANY KIND, either express or implied. See the License for the specific language
 * governing permissions and limitations under the License.
*/
```

Reply:
Should we add TypeScript type for `orientation` prop, or would it be better to rework the API for retrieving gradients based things that make more sense? Orientation is an implementation detail of the way we generate these gradients so passing it around seems odd.

```suggestion
export type OrientationType = ['top', 'left' | 'right'];
```

Reply:
yeah, it did seem a bit specific, which is why I hadn't typed it
do you have an idea for something that would make more sense?